src/Microsoft.DotNet.Build.Tasks.net45/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "Newtonsoft.Json": "7.0.1",
-    "NuGet.Versioning": "3.4.0-beta-488",
+    "NuGet.Versioning": "3.5.0-rc-1220",
```

Comment:
Looks like this change caused a bunch of dependency conflicts because this new version of NuGet depends on `NETStandard.Library version 1.5.0-rc2-24022` but the packages we use are `23923` version instead. To repro this, simply grab latest code of corefx and run init-tools, and you will get dependency conflicts similar to the following:

```
C:\Users\joperezr\Desktop\repo\corefx\packages\Microsoft.DotNet.BuildTools\1.0.25-prerelease-00330-01\lib\tool-runtime\project.json(18,57): warning NU1012: Dependency conflict. NETStandard.Library 1.5.0-rc2-24022 expected Microsoft.NETCore.Platforms >= 1.0.1-rc2-24022 but got 1.0.1-rc2-23923
C:\Users\joperezr\Desktop\repo\corefx\packages\Microsoft.DotNet.BuildTools\1.0.25-prerelease-00330-01\lib\tool-runtime\project.json(19,55): warning NU1012: Dependency conflict. NETStandard.Library 1.5.0-rc2-24022 expected Microsoft.NETCore.Runtime >= 1.0.2-rc2-24022 but got 1.0.2-rc2-23923
C:\Users\joperezr\Desktop\repo\corefx\packages\Microsoft.DotNet.BuildTools\1.0.25-prerelease-00330-01\lib\tool-runtime\project.json(21,38): warning NU1012: Dependency conflict. System.Diagnostics.Process 4.1.0-rc2-24022 expected Microsoft.Win32.Registry >= 4.0.0-rc2-24022 but got 4.0.0-rc2-23923
C:\Users\joperezr\Desktop\repo\corefx\packages\Microsoft.DotNet.BuildTools\1.0.25-prerelease-00330-01\lib\tool-runtime\project.json(38,44): warning NU1012: Dependency conflict. NETStandard.Library 1.5.0-rc2-24022 expected System.Console >= 4.0.0-rc2-24022 but got 4.0.0-rc2-23923
C:\Users\joperezr\Desktop\repo\corefx\packages\Microsoft.DotNet.BuildTools\1.0.25-prerelease-00330-01\lib\tool-runtime\project.json(30,53): warning NU1012: Dependency conflict. NuGet.RuntimeModel 3.5.0-rc-1220 expected System.Dynamic.Runtime >= 4.0.11-rc2-24022 but got 4.0.11-rc2-23923
C:\Users\joperezr\Desktop\repo\corefx\packages\Microsoft.DotNet.BuildTools\1.0.25-prerelease-00330-01\lib\tool-runtime\project.json(31,51): warning NU1012: Dependency conflict. NETStandard.Library 1.5.0-rc2-24022 expected System.IO.Compression >= 4.1.0-rc2-24022 but got 4.1.0-rc2-23923
C:\Users\joperezr\Desktop\repo\corefx\packages\Microsoft.DotNet.BuildTools\1.0.25-prerelease-00330-01\lib\tool-runtime\project.json(31,51): warning NU1012: Dependency conflict. NuGet.Packaging 3.5.0-rc-1220 expected System.IO.Compression >= 4.1.0-rc2-24022 but got 4.1.0-rc2-23923
... (Many More)
```

@ericstj would it be ok to just simply move the framework we depend on to 24022 to match NuGet instead?